test/games/strategy/triplea/ui/TestRoute.java

```diff
@@ -0,0 +1,64 @@
+package games.strategy.triplea.ui;
```

Comment:
Problem here, currently all of those methods are static and mockito does not support static mocks...powermockito Could help out there - or a complicated refactoring